ownership-chain/node/src/chain_spec.rs

```diff
@@ -181,7 +181,7 @@ fn testnet_genesis(
 		},
 		collator_selection: laos_ownership_runtime::CollatorSelectionConfig {
 			invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
-			candidacy_bond: EXISTENTIAL_DEPOSIT * 16,
+			candidacy_bond: laos_ownership_runtime::ExistentialDeposit::get() * 16,
```

Comment:
we are requiring 0 bond for candidacy to be a collator? This should be a non-zero value, IMO

Reply:
please open an issue on the topic

Reply:
I think, if you are changing something in the code, then everything that depends on this constant should be addressed as well. I suggest at least creating a new constant that does not depend on the `ED` anymore. Value can be calculated with previous non-zero value of ED

```
16 * PREVIOUS_ED
```